lms/static/sass/shared/_header.scss

```diff
@@ -906,3 +882,27 @@
     color: $link-color !important;
   }
 }
+
+.doc-link {
+    position: relative;
```

Comment:
Are all these rules necessary too? I'd rather keep this simple and have it adopt the regular styling from the page where possible.